### PR TITLE
Updated class for result card image in teams and mentor search result

### PR DESCRIPTION
**Pull request by viviancan**
Refs #4344 

A bug was discovered in the result card for the team and mentor search. The team info would sometimes be hidden in the result card depending on image size. This change updates the class for the result card image to avoid this issue. Now all images regardless of size, will fit at the top of the result card. There is also a lot of auto formatting from the linter.

### CODE REVIEW

File: app/javascript/components/ResultCard.vue [hunk 1]
```diff
@@ -1,13 +1,20 @@
 <template>
-  <div class="card-result">
+  <div class="card-result bg-gray-50">
```
Comment (viviancan) [inline app/javascript/components/ResultCard.vue:2]: Added a light bg color to the card so they wouldn't blend into the background

File: app/javascript/components/ResultCard.vue [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
     <div class="relative">
-      <img :src="cardImage" :class="coverImageClass(coverImage)" @error="imgBroken" />
+      <img
+        :src="cardImage"
+        class="object-contain h-48 w-96"
```
Comment (viviancan) [inline app/javascript/components/ResultCard.vue:6]: Added these values for the images

File: app/javascript/components/ResultCard.vue [hunk 3]
```diff
@@ -19,63 +26,64 @@
...
-        let bgs = {
-          'team': { on: 'bg-blue-600', off:'bg-pink-600' },
-          'virtual': { on: 'bg-pink-600', off:'bg-blue-600' }
-        }
-        
-        let bg = flag ? bgs[type].on : bgs[type].off
-        
-        return `list-badge ${bg} left-arrow uppercase`
-      },
-      coverImageClass(flag) {
```
Comment (viviancan) [inline app/javascript/components/ResultCard.vue:71]: Removed this method

File: app/javascript/components/ResultCard.vue [hunk 4]
```diff
@@ -1,13 +1,20 @@
...
-      <img :src="cardImage" :class="coverImageClass(coverImage)" @error="imgBroken" />
+      <img
+        :src="cardImage"
+        class="object-contain h-48 w-96"
+        @error="imgBroken"
+      />
 
       <div class="card-photo-placeholder hidden" :id="imgPlaceholderId">
-        <p class="text-xl font-bold text-black-200">No picture<p/>
+        <p class="text-xl font-bold text-black-200">No picture</p>
```
Comment (shaun-technovation) [inline app/javascript/components/ResultCard.vue:11]: Good catch here!

**Review by shaun-technovation — APPROVED** [review]
Looks good to me! 👍 🎉 🚀

**Pull request merged**